packages/components/src/components/guild-navbar-logo.stories.tsx

```diff
@@ -21,5 +21,5 @@ export const Default: StoryObj<typeof GuildUnifiedLogo> = {
 };
 
 export const Hive: StoryObj = {
-  render: () => getNavbarLogo(HiveCombinationMark, 'Hive', ''),
+  render: () => getNavbarLogo(<HiveCombinationMark />, 'Hive', ''),
```

Comment:
we cannot pass function for server components to client, so `getNavbarLogo` now accept react node